python/bind_detail.h

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2019 Scipp contributors (https://github.com/scipp)
+/// @file
+/// @author Neil Vaytet
+#ifndef SCIPP_BIND_DETAIL_H
+#define SCIPP_BIND_DETAIL_H
```

Comment:
`SCIPP_PYTHON_BIND_DETAIL_H`

Also, why is this named `bind_detail`? You are not binding anything here.